ctf/README.md

```diff
@@ -0,0 +1,47 @@
+### CFS Test Framework
+
+This repository contains the scripts and plugins used to validate the cFE flight software.
+
+
+### Getting Started
+
+To get started, clone the CTF repository using:
+
+git clone git@aetd-git.gsfc.nasa.gov:gateway_cert/cfs_gw.git
+cd cfs_gw
+git submodule update --init
+
+Note that the CTF repository must be cloned at the same directory level as the cFS repository.
```

Comment:
We shouldn't include the internal steps for the open source instructions.  Open source steps would be to clone cFS and clone CTF from public repos.

Reply:
And CTF hasn't been open source released yet... but leave as a TBD?